### PR TITLE
[VPUX] - Get rid of dependency to inference_engine_legacy

### PR DESCRIPTION
**Pull request by DariaMityagina**

### CODE REVIEW

File: src/core/CMakeLists.txt [hunk 1]
```diff
@@ -47,7 +47,8 @@ source_group("include" FILES ${PUBLIC_HEADERS})
 add_library(ov_core_dev INTERFACE)
 add_library(openvino::core::dev ALIAS ov_core_dev)
 
-target_include_directories(ov_core_dev INTERFACE $<BUILD_INTERFACE:${OpenVINO_SOURCE_DIR}/src/core/dev_api>)
+target_include_directories(ov_core_dev INTERFACE $<BUILD_INTERFACE:${OpenVINO_SOURCE_DIR}/src/core/dev_api>
+                                       INTERFACE $<BUILD_INTERFACE:${OpenVINO_SOURCE_DIR}/src/common/transformations/include>)
```
Comment (ilya-lavrenov) [inline src/core/CMakeLists.txt:51]: we already have this in master:
https://github.com/openvinotoolkit/openvino/blob/6f3b37b2fabf6409beee6e5f1440f4f294163972/src/core/CMakeLists.txt#L50-L54

**Comment by ilya-lavrenov** [comment]
In general the PR does not make sense.
We don't extend existing operations.

**Comment by DariaMityagina** [comment]
> In general the PR does not make sense. We don't extend existing operations nor change the semantic.

- The purpose of this PR is to get rid of legacy dependencies in VPUX plugin, since legacy has been moved to GNA plugin
- A temporary solution was to enable INTEL_GNA=ON when building the VPUX plugin with the Openvino master branch. But this is not acceptable. But this is unacceptable. That's why I needed to extend the existing operations and use them in the VPUX plugin instead of the legacy ones.

**Comment by ilya-lavrenov** [comment]
> > In general the PR does not make sense. We don't extend existing operations nor change the semantic.
> 
> * The purpose of this PR is to get rid of legacy dependencies in VPUX plugin, since legacy has been moved to GNA plugin
> * A temporary solution was to enable INTEL_GNA=ON when building the VPUX plugin with the Openvino master branch. But this is not acceptable. But this is unacceptable. That's why I needed to extend the existing operations and use them in the VPUX plugin instead of the legacy ones.

I understand your pain, but such PR will not be accepted.
You need to introduce new operations in VPUX internal opset and use them instead of IE Legacy operations.

**Comment by DariaMityagina** [comment]
> > > In general the PR does not make sense. We don't extend existing operations nor change the semantic.
> > 
> > 
> > 
> > * The purpose of this PR is to get rid of legacy dependencies in VPUX plugin, since legacy has been moved to GNA plugin
> > * A temporary solution was to enable INTEL_GNA=ON when building the VPUX plugin with the Openvino master branch. But this is not acceptable. But this is unacceptable. That's why I needed to extend the existing operations and use them in the VPUX plugin instead of the legacy ones.
> 
> I understand your pain, but such PR will not be accepted. You need to introduce new operations in VPUX internal opset and use them instead of IE Legacy operations.

Okay, I see. Will try to do that. Thanks!

**Pull request closed without merging**